src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -99,6 +99,9 @@ export class NetCoreTaskHelper implements TaskHelper {
         options.appProject = options.appProject || await NetCoreTaskHelper.inferAppProject(context.folder); // This method internally checks the user-defined input first
         options.platformOS = options.platformOS || 'Linux';
 
+        // If there's exactly one port and it's 80, then set configureSsl to false, otherwise leave it undefined

```

Comment:
Will the `configureSsl: undefined` show up in tasks.json after this change? If yes, can we avoid that? In general it is preferable to avoid explicit configuration if defaults are used.

Reply:
No, fortunately! It does exactly what we want:

When you pass in `configureSsl: undefined`:
```json
"netCore": {
    "appProject": "${workspaceFolder}/MVC3.csproj",
    "enableDebugging": true
}
```

And `false`:
```json
"netCore": {
    "appProject": "${workspaceFolder}/MVC3.csproj",
    "enableDebugging": true,
    "configureSsl": false
}
```